endpoint/test_endpoint_development.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+TARGET_VERSION="3.4.0"
```

Comment:
Move Legato AF specific configurations into separated shell script and use "source" to include it.